src/components/Select.js

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash');
```

Comment:
Rather than pulling in the entire library for one method you can pull just `isEqual` by doing `const _isEqual = require('lodash/isEqual')` if we're on the latest version of lodash and `const _isEqual = require('lodash/lang/isEqual')` if we're not.